lib/vanagon/component.rb

```diff
@@ -290,6 +290,10 @@ def get_build_dir
       end
     end
 
+    def get_dependency_hash
+      { name => { 'version' => version, 'ref' => options[:ref] }.delete_if { |k, v| !v } }
```

Comment:
Haha we could argue about Ruby all day 😄 

I've succumbed to the state of the code, so I think we've resolved this.

Have a good weekend @mckern 

I look forward to seeing you at the carts again soon